Readme.md

```diff
@@ -326,17 +340,48 @@ mod1       | some description which happens to be far larger than the max | 0.0.
 module-two | another description larger than the max                      | 0.2.0
 ```
 
-### Do not show headers
+### Control Header Display
 
-Prevent columns headers from showing with:
+Control whether column headers are displayed by using the `showHeaders` option.
 
 ```javascript
-
 var columns = columnify(data, {
   showHeaders: false
 })
 ```
 
+### Transforming Column Data and Headers
+If you need to modify the presentation of column content or heading content there are two useful options for doing that: `dataTransform` and `headerTransform`. Both of these take a function and need to return a valid string.
+
+```javascript
+var columns = columnify([{
+    name: 'mod1',
+    description: 'SOME DESCRIPTION TEXT.'
+}, {
+    name: 'module-two',
+    description: 'SOME SLIGHTLY LONGER DESCRIPTION TEXT.'
+}], {
+    dataTransform: function(data) {
+        return data.toLowerCase();
+    },
+    config: {
+        name: {
+            headingTransform: function(heading) {
+              heading = "module " + heading;
+              return "\u001b[32m" +  heading.toUpperCase() + "\u001b[39m";// green header
```

Comment:
Since the color does not show in the output here, I'd consider using a different style to demonstrate it. Maybe `return '*' + heading.toUpperCase() + '*';`. Also, this could make the line short enough to fit the github readme page without horizontal scrolling.